pkgs/os-specific/linux/sgx/psw/default.nix

```diff
@@ -131,6 +131,31 @@ stdenv.mkDerivation rec {
     rmdir $sgxPswDir || (echo "Error: The directory $installDir still contains unhandled files: $(ls -A $installDir)" >&2 && exit 1)
   '';
 
+  stripDebugList = [
+    # Only strip the host libraries. If we don't strip, we accidentally pull in
+    # `gcc`...
+    #
+    # We need to avoid stripping the enclave ".signed.so" files, since that
+    # would modify the enclave measurement (hash).
+    #
+    # TODO(phlip9): put the enclave libraries in a separate directory or tell
+    # strip to ignore all "*.signed.so".
```

Comment:
Is there a way to make this derivation fail to build when we fail to strip a binary here?

Reply:
Or alternatively: could we copy the files in after the stripping?

Reply:
Went with the second approach here: move the enclaves to a $TMPDIR during fixup, then move them back afterwards. Should be significantly more robust.